### PR TITLE
Display error alert box on supp app if form is valid

### PR DESCRIPTION
**Pull request by chrisdolendo**
PT Story: https://www.pivotaltracker.com/story/show/164615215

### CODE REVIEW

File: app/javascript/components/applications/application_form/PaperApplicationForm.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ class PaperApplicationForm extends React.Component {
                     <button className='primary radius margin-right save-btn' type='submit' onClick={() => this.saveSubmitType('Save', form)} disabled={loading}>
                       {loading ? 'Saving…' : 'Save'}
                     </button>
-                    <button className='primary radius' type='submit' onClick={() => this.saveSubmitType('SaveAndNew')} disabled={loading}>
+                    <button className='primary radius' type='submit' onClick={() => this.saveSubmitType('SaveAndNew', form)} disabled={loading}>
```
Comment (chrisdolendo) [inline app/javascript/components/applications/application_form/PaperApplicationForm.js:116]: I believe this button also needs form passed through as well so I added it, not part of the feature but a fix.

File: app/javascript/components/supplemental_application/SupplementalApplicationContainer.js [hunk 2]
```diff
@@ -197,7 +195,26 @@ class SupplementalApplicationContainer extends React.Component {
               <DemographicsSection />
               <ScrollableAnchor id={'status-history-section'}><div><StatusHistorySection /></div></ScrollableAnchor>
               <div className='padding-bottom--2x margin-bottom--2x' />
-              <ActionButtons loading={loading} />
+              <div className='button-pager'>
```
Comment (chrisdolendo) [inline app/javascript/components/supplemental_application/SupplementalApplicationContainer.js:198]: This was `<ActionButtons/`> but I needed to access the `state`, so I included it in this main component and deleted `<ActionButtons/`> .

**Comment by akegan** [comment]
@chrisdolendo this is ready to be rebased onto master! 🎉

**Comment by chrisdolendo** [comment]
Reopened in https://github.com/Exygy/sf-dahlia-lap/pull/259

**Pull request closed without merging**